x/contracts/runtime/util_test.go

```diff
@@ -198,11 +198,24 @@ func (t *testRuntime) CallContract(contract codec.Address, function string, para
 }
 
 func newTestRuntime(ctx context.Context) *testRuntime {
+	callContext := NewRuntime(
+		NewConfig(),
+		logging.NoLog{}).WithDefaults(CallInfo{Fuel: 1000000000})
+
+	r := callContext.r
+
+	if r.linkerNeedsInitialization {
+		linker, err := r.hostImports.createLinker(r)
+		if err != nil {
+			panic(err)
+		}
+		r.linker = linker
+		r.linkerNeedsInitialization = false
+	}
```

Comment:
Don't need this lazy flag anymore. Not sure why it was ever done this way, the host-imports _should_ be loaded when the runtime is initialized.